### ISSUE TITLE
PasswordValidationOnSignUp component conflict with backend->Add User

### ISSUE THREAD

**Issue by githubertus** [opensource-socialnetwork/opensource-socialnetwork#1515]
PasswordValidationOnSignUp component needs password lenght of _8_ mininum
But the length check isn't only used on signup, but also
1. on change of Account Settings
2. on Add User from admin backend

Problem:
running either of both parts currently displays the old DEFAULT error message:
"Password must be more than _5_ characters"
which of course is irritating.
So the component has to overwrite the default message like
```
	// overwrite Ossn default length of 6 with component defined length -1
	'password:error' => 'Password must be more than 7 characters.',

```

**Comment by lianglee** [comment]
Agree, i guess we need to modify the password validation component with new string.

**Comment by lianglee** [comment]
Moving to https://github.com/lianglee/PasswordValidation

**Comment by lianglee** [comment]
https://github.com/opensource-socialnetwork/PasswordValidation
